package.json

```diff
@@ -31,7 +31,9 @@
   },
   "author": "Yassine ELOUAFI <yelouafi@gmail.com>",
   "contributors": [
-    "Mateusz Burzyński <mateuszburzynski@gmail.com> (https://github.com/Andarist)"
+    "Mateusz Burzyński <mateuszburzynski@gmail.com> (https://github.com/Andarist)",
+    "Mikhail Shustov (https://github.com/restrry)",
```

Comment:
feel free to add my email -  <restrry@gmail.com> 
anyway it's exposed by Github API. I used it as a workaround to contact some people  😄